media/css/firefox/hello/index-2016.less

```diff
@@ -0,0 +1,821 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+#outer-wrapper {
+    background: @mozIDBlue;
+}
+
+#wrapper {
+    width: 100%;
+    padding: 0;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    color: #fff;
+    text-shadow: none;
+}
+
+.container {
+    width: @widthDesktop - (@baseLine * 2);
+    padding: (@baseLine * 2) 0;
+
+    @media only screen and (max-width: @breakDesktop) {
+        width: @widthTablet - (@gridGutterWidth * 2);
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        width: @widthMobileLandscape;
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        width: @widthMobile - (@gridGutterWidth * 2);
+    }
+}
+
+.try-hello-button,
+.try-hello-button:link,
+.try-hello-button:visited {
+    .inline-block;
+    padding: 12px (@baseLine * 2) 12px 85px;
+    margin-bottom: @baseLine;
+    .open-sans-light;
+    .font-size(30px);
+    color: #fff;
+    text-align: center;
+    .at2x('/media/img/firefox/hello/2016/hello-smiley.png', 40px, 38px);
+    background-position: 32px center;
+    background-repeat: no-repeat;
+    background-color: #005189;
+    border-radius: 6px;
+    border: none;
+    cursor: pointer;
+    .transition(background-color .1s ease-in-out);
+
+    @media only screen and (max-width: @breakDesktop) {
+        .font-size(26px);
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        .font-size(24px);
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        .font-size(20px);
+    }
+
+    &:hover,
+    &:focus {
+        text-decoration: none;
+        background-color: lighten(#005189, 3%);
+        color: #fff;
+    }
+}
+
+#fxfamilynav-header {
+    .try-hello-button,
+    .nav-download-button {
+        .open-sans;
+        .font-size(@smallFontSize);
+        padding: 12px (@baseLine / 2);
+        background: #005189;
+        text-transform: none;
+
+        &:hover,
+        &:focus {
+            background: lighten(#005189, 3%);
+        }
+    }
+}
+
+html[lang^='en'] {
+    #fxfamilynav-header {
+        .try-hello-button,
+        .nav-download-button {
+            text-transform: uppercase;
+        }
+    }
+}
+
+/**
+ * Conditional messaging
+ * ---------------------
+ */
+
+.no-js {
+    .non-fx {
+        display: block;
+    }
+
+    .nav-download-button,
+    .hello-wordmark {
+        display: inline-block;
+    }
+
+    .fx-up-to-date,
+    .fx-out-of-date,
+    .try-hello-button,
+    .mobile {
+        display: none;
+    }
+
+    .download-dumb {
+        text-align: center;
+    }
+}
+
+.js {
+    .fx-up-to-date,
+    .fx-out-of-date,
+    .non-fx,
+    .mobile,
+    .download-button,
+    .hello-wordmark,
+    .nav-download-button,
+    .try-hello-button, {
```

Comment:
Nit: I'm guessing the preprocessor is fine with this but, the last item should not have a `,`